app/src/pages/Dashboard/Groups/Views/Studies.vue

```diff
@@ -567,28 +567,24 @@
         this.showModal = true;
       },
       createStudy() {
-        if (!this.newStudy.name || !this.newStudy.description) {
-          Message.error('Please enter a name and description for your new study');
-          return;
-        }
 
-        this.creatingStudy = true;
+            if (!this.newStudy.name || !this.newStudy.description) {
+                Message.error('Please enter a name and description for your new study');
+                return;
+            }
 
-        this.$store
-          .dispatch('study/createStudy', {
-            title   : this.newStudy.name,
-            content : this.newStudy.description,
-            author  : this.user.me.id,
-            status  : 'private',
-            sc_group: [this.groupOrgID + ''],
```

Comment:
The sc_group taxonomy is critical for linking a study to an organization church. We need to keep this value if the current group belongs to an organization.